plugins/notifications/src/components/NotificationsPage/NotificationsPage.tsx

```diff
@@ -24,6 +24,15 @@ import {
 import Grid from '@material-ui/core/Grid';
 import { ConfirmProvider } from 'material-ui-confirm';
 import { useSignal } from '@backstage/plugin-signals-react';
+import { useTranslationRef } from '@backstage/core-plugin-api/alpha';
+import { notificationsTranslationRef } from '../../translation';
+
+const TableTitleKeys = {
+  all: 'notificationsPage.tableTitle.all' as const,
+  saved: 'notificationsPage.tableTitle.saved' as const,
+  unread: 'notificationsPage.tableTitle.unread' as const,
+  read: 'notificationsPage.tableTitle.read' as const,
+} as const;
 
```

Comment:
```suggestion
const TableTitleKeys = {
  all: 'notificationsPage.tableTitle.all' ,
  saved: 'notificationsPage.tableTitle.saved' ,
  unread: 'notificationsPage.tableTitle.unread' ,
  read: 'notificationsPage.tableTitle.read' ,
} as const;

```
@ciiay I just want to check if we can remove `as const` from each entries.